README.md

```diff
@@ -28,33 +28,33 @@ Certificate Generator aims to automatically generate certificates based on the d
 * _**For Candidates:**_
     i. The dashboard will contain thumbnails of all the certificates received by the student.
     ii. There will be an option to view a particular certificate.
-    iii. The candidate will also be allowed to share the certificate.
+    iii. The candidate will also be allowed to share the certificate with others on any social platform.
 
 
 
 
 
-### Development Steps
+### Development Steps:
 
 Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
 
-#### Code scaffolding
+#### Code scaffolding:
 
 Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
-#### Build
+#### Build/:
 
 Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
 
-#### Running unit tests
+#### Running unit tests:
```

Comment:
for heading adding ":" is not mandatory, even official angular repo follows the same pattern. They do not add :